docs/packaging.md

```diff
@@ -158,154 +89,74 @@ Note that the configuration is still in YAML format, but reduced in scope.
 
-After installing the `aziot-edge` package, run `iotedge init` to interactively set up the configuration. It performs the initialization for both the IS+KS+CS+TPMS components installed by the `aziot-identity-service` package and the MR component installed by the `aziot-edge` package.
+For example, if you have a module process, the process needs to interact with the IS to get its module identity, with the KS to be able to use its module identity key to create a SAS token or for mTLS, and with the CS in order to use its module identity certificate for mTLS. So you should add the user that your module runs as to the `aziotid`, `aziotks` and `aziotcs` groups.
 
-If the config files already exist, `iotedge init` will fail instead of overwriting them. Using `iotedge init --force` will force it to overwrite them.
 
-## Updating from `iotedge` to `aziot-edge`
+## Home directories
 
-```sh
-apt remove iotedge libiothsm-std
-
-apt install aziot-edge
-
-iotedge init
+```
+/var/lib/aziot/
+├── certd/
+│   └── certs/
+├── identityd/
+├── keyd/
+│   └── keys/
+└── tpmd/
 ```
 
-The user must remove the existing `iotedge` and `libiothsm-std` packages before installing the `aziot-edge` package (or even the `aziot-identity-service` package). We do not want a situation where the services from both packages are running at the same time. They would step over each other trying to provision the device and manage Docker modules. We enforce mutual exclusivity between the packages by having them conflict with each other so that the distribution's package manager does not allow them both to be installed at the same time.
-
-The `iotedge init` automatically detects when the configuration of the old IoT Edge installation is available and prompts whether to create the new configuration based on the old. Agreeing to import the old configuation does not remove it.
+`/var/lib/aziot/certd/certs` stores any certificates imported or generated by CS.
 
-### Configuration
+`/var/lib/aziot/keyd/keys` stores any keys imported or generated by KS that could not be stored in an HSM via PKCS#11. (This is the case if the HSM doesn't support that particular kind of key, or if KS was not configured to use PKCS#11 at all.)
 
-Alternatively, a non-interactive initialization that attempts to import the old configuration from `iotedge` can be done using the `import` subcommand.
 
-```sh
-iotedge init import
-```
+## Package dependencies
 
-- Device provisioning method is parsed from `iotedge/config.yaml` and translated into the provisioning information in `identityd/config.toml`, `keyd/config.toml`, `certd/config.toml`, `tpmd/config.toml` and `edged/config.yaml`. For example, in case of manual-symmetric-key provisioning, the SAS key will be imported as a preloaded key in `keyd/config.toml`, and `identityd/config.toml` will be updated to use manual provisioning with a reference to the key ID.
+The binaries and libraries shipped in this package only depend on OpenSSL.
```

Comment:
Might be worth calling out which version of openssl.

Reply:
Done.